app/modules/asset_groups/models.py

```diff
@@ -58,7 +57,8 @@ class DataProcessed(str, enum.Enum):
         sightings = 2
         complete = 3
 
-    def __init__(self, request_json):
+    def __init__(self, request_json, existing_dir=None):
```

Comment:
discussed in slack -- `existing_dir` here is no longer needed so will be removed.